app/Http/Controllers/Api/WorkController.php

```diff
@@ -17,7 +17,7 @@ public function index(Request $request)
 
         // search
         if ($request->filled('q')) {
-            $works->where('name', 'like', '%' .$request->input('q') . '%');
+            $works->whereIn('id', Work::search($request->query('q'))->keys());
```

Comment:
this wouldn't work the same way. it supposed to be "quick filter"
what about:
```suggestion
            $works->whereIn('id', Work::search("*{$request->query('q')}*")->keys());
```
? (same as you did in `app/Http/Controllers/Api/ArchitectController.php`